package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@descript/draft-js",
   "description": "A React framework for building text editors.",
-  "version": "0.11.6-descript.17",
+  "version": "0.11.6-descript.22",
```

Comment:
This is using `yarn` and because we're on `pnpm` now, I can't use `yarn link` for local testing. I'm sure there's a pnpm equivalent, but this was the easiest way (and I didn't have to convert this repo to pnpm).